carma-messenger-core/docker/checkout.bash

```diff
@@ -40,3 +40,8 @@ done
 git clone --depth=1 https://github.com/usdot-fhwa-stol/v2x-ros-conversion.git ${dir}/src/v2x-ros-conversion --branch "develop"
 git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ${dir}/src/CARMAMsgs --branch $BRANCH
 git clone https://github.com/usdot-fhwa-stol/carma-utils.git ${dir}/src/CARMAUtils --branch $BRANCH
+
+
+# Install dependencies
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Comment:
NIT: I don't think you need this. I think you should just be able to call
```bash
./install_dependencies.sh ....
```

Reply:
Seems like this is required when building the Dockerfile because the checkout script is not called from its parent directory so "./install_dependencies.sh" doesn't work.